### ISSUE TITLE
Releasing a note doesn't trigger a notecut effect, holding a note does not cause sustain

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kometbomb/klystrack#95]
```
What steps will reproduce the problem?
1.Create an instrument with sustain
2.Play notes to test instrument sound

--What is the expected output? What do you see instead?
I would expect for holding a note down, for example holding "Q" on the keyboard 
to play a C, to play the note while I hold it down, and for letting up on the 
key to trigger a note cut.  This is how you test the sound of an instrument 
you're creating.  Instead if there is sustain, pressing a note at all will 
cause the note to continue playing until you switch to the pattern editor and 
click stop.  In addition, holding down a note will cause a note to repeatedly 
re-trigger in rapid succession rather than sustain a note.  These things make 
for a difficult to use instrument editor.

--What version of the product are you using? On what operating system?
v 1.6 on Windows 7

--Please provide any additional information below.

Basically, it would be nice if playing notes in the instrument editor would 
mimic the instruments actual behavior in the pattern editor.
```

Original issue reported on code.google.com by `overleve...@gmail.com` on 30 Jan 2013 at 8:07
